crt_portal/cts_forms/templates/partials/policy.html

```diff
@@ -1,30 +0,0 @@
-{% load i18n %}
```

Comment:
We still need this on the form pages- just confirmed with Dan

Reply:
Ahh good call, that makes alot of sense! Re-added.